apps/financial-aid/backend/src/app/guards/application.guard.ts

```diff
@@ -31,7 +31,10 @@ export class ApplicationGuard implements CanActivate {
         return false
       }
     } else if (user.service === RolesRule.OSK) {
-      if (application.nationalId !== user.nationalId) {
+      if (
+        application.nationalId !== user.nationalId &&
```

Comment:
Shouldn't this be || ?

Reply:
No because it is !==, we are checking if application national id is not the same as the user and not the same as the spouse, then return forbidden